packages/message-box/src/message-box.type.ts

```diff
@@ -1,7 +1,7 @@
 import type { VNode } from 'vue'
 
 export type Action = 'confirm' | 'close' | 'cancel'
-export type MessageType = 'success' | 'warning' | 'info' | 'error'
+export type MessageType = '' | 'success' | 'warning' | 'info' | 'error' | 'prompt' | 'alert' | 'confirm'
```

Comment:
'prompt' | 'alert' | 'confirm' these are action types not message types, give them an another type declaration

Reply:
or why we mix types and actions?

Reply:
Affected by the original code, because the other three type boxes passed their box type to `type`.

It has been corrected.